example/src/main/java/com/vimeo/android/networking/example/vimeonetworking/TestAccountStore.java

```diff
@@ -38,19 +39,18 @@ public VimeoAccount loadAccount() {
         return AccountPreferenceManager.getClientAccount();
     }
 
-    @Deprecated
     @Override
-    public void saveAccount(VimeoAccount vimeoAccount, String email, String password) {
-        saveAccount(vimeoAccount, email);
+    public void saveAccount(@NonNull VimeoAccount vimeoAccount) {
+        AccountPreferenceManager.setClientAccount(vimeoAccount);
     }
 
     @Override
-    public void saveAccount(VimeoAccount vimeoAccount, String email) {
+    public void saveAccount(@NonNull VimeoAccount vimeoAccount, @NonNull String email) {
```

Comment:
I think if we were to use the second parameter then we would need a lot more demo code than we have; I think the demo could be improved upon greatly, but I also don't think this is the time to do it. I will make sure the name matches though.